.github/ISSUE_TEMPLATE/2_documentation.yml

```
@@ -0,0 +1,33 @@
+name: Documentation
+description: Create a documentation related issue.
+labels:
+  - type::documentation
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > **Note**
+        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.
+        💐 Thank you for helping to make conda better. We would be unable to improve conda without our community!
```

While the admonition syntax doesn't render correctly in issue forms we should use it for the day that it does. To render something like:

> **Note**
> Body

We need:

```
> **Note**
> Body
```

I also see that this text is copied verbatim from the feature template so I've made some modifications to preserve spacing and intent.

```suggestion
        > **Note**
        > Documentation requests that are incomplete or missing information may be closed as inactionable.

        Since there are already a lot of open issues, please also take a moment to search existing ones to see if your bug has already been reported. If you find something related, please upvote that issue and provide additional details as necessary.

        💐 Thank you for helping to make conda better. We would be unable to improve conda without our community!
```